runtime/parser/parse_metrics_view.go

```diff
@@ -704,6 +786,168 @@ func (p *Parser) parseMetricsView(node *Node) error {
 	return nil
 }
 
+// parseInlineExplore parses and validates the inline explore definition in a metrics view YAML. It returns true if automatic explore emission should be skipped, false otherwise.
+func (p *Parser) parseInlineExplore(tmp *MetricsViewYAML, mvName string, mvPaths []string) (bool, *Resource, error) {
```

Comment:
can we call this `parseAndInsertInlineExplore`? Since insertion is pretty sensitive, would be good to make it clear